test/planners.jl

```diff
@@ -158,6 +158,15 @@ spec = SymbolicPlanners.BackwardSearchGoal(bw_spec, bw_state)
 
 end
 
+@testset "BiDirectional Planner" begin
+  planner = BidirectionalPlanner(HAdd(), HAddR())
+  sol = planner(blocksworld, bw_state, bw_spec)
```

Comment:
Can we also add a test here to see if the original specification is satisfied?

That is:
```julia
@test is_goal(bw_spec, blocksworld, sol.trajectory[end])
```

Reply:
Certainly, it is added.